pkg/helpers/github_test.go

```diff
@@ -0,0 +1,29 @@
+package helpers
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/pior/dad/pkg/config"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dnaeon/go-vcr/recorder"
+)
+
+func TestLatestRelease(t *testing.T) {
+	r, err := recorder.New("fixtures/github")
```

Comment:
Is it a good idea to spread the fixtures files in package files?
Is there an elegant way to do otherwise?